### PR TITLE
fastifyCookieClearCookie sets maxAge to zero

### LINKED ISSUE #303: clearCookie does not include maxAge, neither allows to pass it as options
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the bug has not already been reported

### Fastify version

5.0

### Plugin version

v10.0.1

### Node.js version

any

### Operating system

Linux

### Operating system version (i.e. 20.04, 11.3, 10)

any

### Description

These are the problematic lines:

https://github.com/fastify/fastify-cookie/blob/ccb90539af657184c07dae983db5bdb7afa8617e/plugin.js#L42-L46

Without `maxAge: 0` at least some of cookie jar implementations (for example https://www.npmjs.com/package/tough-cookie) keep actual cookie with empty value in the storage instead of deleting them (like they do when `maxAge` is set to zero).

Because of this flaw some users had to come up with workarounds [like this](https://github.com/fastify/fastify-secure-session/blob/d322f37b46e911c6a091ff90f20900dd9e1988da/index.js#L278-L284).

### Link to code that reproduces the bug

_No response_

### Expected Behavior

`maxAge` should be set along with `expires` option

### PR DESCRIPTION
**Pull request by aristofun**
Improves devX/compatibility for [some](https://github.com/fastify/fastify-secure-session/blob/d322f37b46e911c6a091ff90f20900dd9e1988da/index.js#L278-L284) users of the library and [some](https://www.npmjs.com/package/tough-cookie) cookie jar implementations. 

Closes https://github.com/fastify/fastify-cookie/issues/303 

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [ ] ~~documentation is changed or added~~
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by gurgunday — CHANGES_REQUESTED** [review]
Temporarily blocking to check something

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fastify/cookie",
-  "version": "10.0.1",
+  "version": "10.0.2",
```
Comment (mcollina) [inline package.json:3]: Please revert this. we'll bump this ourselves.

File: plugin.js [hunk 2]
```diff
@@ -42,7 +42,7 @@ function fastifyCookieClearCookie (reply, name, options) {
   const opts = Object.assign({ path: '/' }, options, {
     expires: new Date(1),
```
Comment (gurgunday) [inline plugin.js:43]:
```suggestion
    expires: new Date(0),
```

File: package.json (hunk 1 above)
Comment (gurgunday) [inline package.json:3]:
```suggestion
  "version": "10.0.1",
```

**Review by gurgunday — APPROVED** [review]
lgtm

**Pull request merged**